client/App.jsx

```diff
@@ -2,13 +2,13 @@ import React from 'react';
 import './App.css';
 import { Provider } from 'react-redux';
 import { PersistGate } from 'redux-persist/integration/react';
-import { Router } from './Router';
+import { RRouter } from './Router';
 import { store, persistor } from './store/store';
 
 export const App = () => (
   <Provider store={store}>
     <PersistGate loading={null} persistor={persistor}>
-      <Router />
+      <RRouter />
```

Comment:
Any reason to do this?

Reply:
It was conflicting with Router from react-router-dom.

Reply:
Could do `import { Router as BrowserRouter } from 'react-router-dom'` - your call here

Reply:
I forgot you could do an alias.  Perfect! fixed. `import { BrowserRouter as BrowserRouter, Switch, Route } from 'react-router-dom';`